src/PublishCodeCovCoverageTask/index.ts

```
@@ -4,6 +4,17 @@ import * as fs from 'fs';
 import * as https from 'https';
 import { execSync } from 'child_process';
 
+// Variable to track if we set the CODECOV_TOKEN
+let tokenWasSetByTask = false;
+
+// Function to clear sensitive environment variables that were set by this task
+function clearSensitiveEnvironmentVariables(): void {
+    if (tokenWasSetByTask && process.env.CODECOV_TOKEN) {
+        console.log('Clearing CODECOV_TOKEN environment variable for security');
+        delete process.env.CODECOV_TOKEN;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)